src/components/articles/ArticleCard.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+
+const ArticleCard = ({ article }) => {
+  return (
+    <div className='card'>
+      <li>
+        <span><strong>{article.title}</strong></span>
+        <p>Author: {article.author}</p>
+        <p><i>Topic: {article.topic}</i></p>
+        <img src="article.article_img_url" alt={article.title} />
```

Comment:
this img src isn't working and we need to use {} to reference the img_url instead of a string.